packages/cli/src/commands/screenshot-diff/screenshot-diff.command.ts

```diff
@@ -305,27 +249,30 @@ const handler: (options: Options) => Promise<void> = async ({
     diffPixelThreshold: pixelThreshold,
   };
 
+  const baseReplayScreenshots = await getScreenshotFiles(baseScreenshotsDir);
+  const headReplayScreenshots = await getScreenshotFiles(headScreenshotsDir);
+
   const results = await diffScreenshots({
     client,
     baseReplayId,
     headReplayId,
     baseScreenshotsDir,
     headScreenshotsDir,
```

Comment:
Given we're now passing the full list of screenshots can the `(base|head)ScreenshotsDir` be dropped as params?